packages/design-tokens/web/components/accordion.json5

```
@@ -3,21 +3,23 @@
         size: {
             item: {
                 header: {
-                    height: { value: '36px' },
+                    height: { deprecated: true, comment: 'DEPRECATED: `component` token will be removed - use `global` token or raw value', value: '36px' },
```

https://github.com/amzn/style-dictionary/blob/86c2c30ba289121f7dc9c28f573d1996dbc4a3a8/examples/advanced/tokens-deprecation/README.md?plain=1#L17
```suggestion
                    height: { deprecated: true, deprecation_comment: 'DEPRECATED: `component` token will be removed - use `global` token or raw value', value: '36px' },
```

---

StyleDictionary outputs only `comment` property, so formatter should be updated.